Xamarin.Forms.Platform.Android.AppLinks/AndroidAppLinks.cs

```diff
@@ -35,19 +35,21 @@ public AndroidAppLinks(Context context)
 			_client.Connect();
 		}
 
-		public void DeregisterLink(IAppLinkEntry appLink)
+		public Task DeregisterLink(IAppLinkEntry appLink)
 		{
 			RemoveFromIndexItemAsync(appLink.AppLinkUri.ToString());
+			return Task.FromResult<object>(null);
```

Comment:
It doesn't matter what result we use, we just need a completed Task to convert this synchronous method to an async one. The result is actually converted from `Task<whatever>` to `Task` so the result is ignored anyway.
The same task/result is used here: https://github.com/xamarin/Xamarin.Forms/blob/master/Xamarin.Forms.Core/NavigationProxy.cs#L201